src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnCache.java

```diff
@@ -169,6 +170,8 @@ public CacheHandle lookup(Spawn spawn, SpawnExecutionContext context)
                 prof.profile(ProfilerTask.REMOTE_DOWNLOAD, "download outputs")) {
               remoteCache.download(
                   result, execRoot, context.getFileOutErr(), context::lockOutputFiles);
+            } catch (DownloadException e) {
+              throw new IOException(e);
```

Comment:
Adding another level of exception wrapping here doesn't sound so good. I already had problems debugging issues with the capabilities call because it was wrapping three levels deep.

Reply:
No disagreement there, this all just hinges on making DownloadException an IOException, which it seems like it should be.